templates/image-builder-clowder.yml

```diff
@@ -14,8 +14,16 @@ objects:
       iqePlugin: image-builder
     deployments:
     - name: service
+      minReplicas: ${{MIN_REPLICAS}}
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG}
+        resources:
+        requests:
+          cpu: "${CPU_REQUEST}"
+          memory: "${MEMORY_REQUEST}"
+        limits:
+          cpu: "${CPU_LIMIT}"
+          memory: "${MEMORY_LIMIT}"
```

Comment:
Shouldn't this be indented?

Reply:
Also, how is this supposed to work? Even when I indent it, looking at the yaml of the pod in OC I see:
```
      containers:
        - resources:
            limits:
              cpu: 300m
              memory: 256Mi
            requests:
              cpu: 30m
              memory: 128Mi
```

Reply:
woops, yea this should be indented. resources should be set on podspec? so unsure why it doesn't get the right values.

Reply:
this is weird, it's clearly in the spec; but it seems like it's not being passed on at all